testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/toolbarModule.tests.js

```diff
@@ -1272,7 +1272,14 @@ testModule('tables', simpleModuleConfig, function() {
         $formatWidgets.each((index, element) => {
             const operationName = TABLE_OPERATIONS[index];
             const isElementExist = Boolean($(element).find(`.dx-icon-${operationName.toLowerCase()}`).length);
+            const expectedDisabledState = operationName === 'insertTable' ? false : true;
+
             assert.ok(isElementExist, `${operationName} item has an related icon`);
+            assert.strictEqual(
+                $(element).hasClass(DISABLED_STATE_CLASS),
+                expectedDisabledState,
+                `${operationName} item should be disabled in case the table is not focused`
```

Comment:
`${operationName} item should ${ expectedDisabledState ? 'not' : '' } be disabled in case the table is not focused`